.github/workflows/main.yml

```diff
@@ -271,6 +288,18 @@ jobs:
           DB_TESTING: ${{ matrix.DB_TESTING }}
         if: ${{ !matrix.COVERAGE }}
 
+      - uses: actions/download-artifact@v3
+        with:
+          name: node_modules
+
+      - name: 'Untar node modules'
+        run: tar -xf node_modules.tar
+
+      - name: Run API tests
+        run: |
+          PWD=$HOME
+          make jstest
```

Comment:
I'm not too keen on having this as a separate step' we integrated everything into a single invocation of prove. You explained how jest can produce tap too, so i was expecting you to integrate with prove, at least for CI purposes.

Reply:
`yath` & `prove` do not have a clue on how to handle `Jest` test files.
We would need to develop another plugin for them to do so.

Reply:
I thought to remember that at least prove would be able to run tests in executables. However, i cannot find any info on that now.